.github/workflows/npm-audit-fix.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branches: ['main', 'master', 'stable29', 'stable28', 'stable27']
```

Comment:
should keep 27

Reply:
This is actually covering branches of the tables app

Reply:
Nope, stable27 is missing now, while info.xml advertises compatibility